11ty-site/pages/app/timeseries.njk

```diff
@@ -1,5 +1,6 @@
 ---
 title: Timeseries
+allowPyScript: true
```

Comment:
Added this since the index.html page was loading pyscript when it didn't need it. So any page requiring pyscript should use this in the front matter